lib/Utils/Backends.pm

```diff
@@ -147,7 +147,7 @@ Returns true if the current instance is running as PowerVM backend 'spvm'
 =cut
 
 sub is_spvm {
-    return check_var('BACKEND', 'spvm');
+    return check_var('BACKEND', 'spvm') || check_var('BACKEND', 'pvm_hmc');
```

Comment:
so "is_spvm" returns true on "spvm" and something else, not a very good idea. Maybe you can rename s/is_spvm/is_pvm/ as you already did a similar change in `lib/bootloader_spvm`